src/main/java/com/dao/DishTypeDAO.java

```diff
@@ -9,17 +9,17 @@
 import java.util.Optional;
 
 
-public class DishTypeDAO {
+public class DishTypeDAO implements RegularDAO<DishType> {
```

Comment:
we decided not to create dao for dish types, didnt we?

Reply:
DishTypeDAO already was in master branch. Not responsible for that.